### DISCUSSION TITLE
Getting the script to run on new subtitle download

### DISCUSSION THREAD

**Discussion by RedRubble** [KBlixt/subcleaner#35] [Q&A] [answered]
Hi folks,
I've been setting this script up and it works great! Used it to clean my entire library. But I'm having trouble getting it to work upon new download. There is just no log when I download a new sub, so I'm not sure if its even running.
In Bazarr, I've got 'Custom Post-Processing' setup, with `python3 /subcleaner/subcleaner.py "{{subtitles}}"` in the box.
I'm using unraid, and have `opt/subcleaner/`'s container path set to `/subcleaner/`.
Upon import though, there is no log. And when I run `python3 /subcleaner/subcleaner.py "{{subtitles}}"`, replacing `"{{subtitles}}"` with the subtitle path, it works.

Any ideas what I'm doing wrong here?

**Comment by KBlixt** [comment]
Maybe you could send an image of the unraid setup page? Just to verify all the paths.

If it checks out it could also be a permission issue.

**Comment by RedRubble** [comment]
![Screenshot 2023-04-15 at 14 52 18](https://user-images.githubusercontent.com/69933306/232183765-a742ee05-e8f0-4c59-8ccc-d9e91f0fffce.jpg)
Sure thing. Bit of a noob so most likely I've got config setup wrong somewhere along the way

**Reply by KBlixt** [reply]
sorry, totally forgot about this, did you sort it out?

**Reply by RedRubble** [reply]
Never managed to get it sorted. I tried setting up an autoscript but that to is a little janky. This is what I've got in Bazarr:

![Screenshot 2023-05-21 at 12 17 14](https://github.com/KBlixt/subcleaner/assets/69933306/fc5c501c-5c5e-43e1-ac98-513dc617d1f3)

**Reply by KBlixt** [reply]
Ok, you have an extra / in the beginning there in the command.

**Reply by RedRubble** [reply]
My bad, bad screenshot. / is still removed and I'm downloading subtitles and the log doesn't report anything..

**Reply by KBlixt** [reply]
Have you tried with the official bazarr image?

**Comment by RedRubble** [comment, marked as answer]
Just putting this here, times have changed and I'm a better debugger now. Got back to checking if I could figure it out now it's been a year, it was literally just a permission issue! So all fixed :)
